src/binder/bind/bind_copy.cpp

```diff
@@ -37,7 +44,8 @@ std::unique_ptr<BoundStatement> Binder::bindCopyClause(const Statement& statemen
         }
     }
     return std::make_unique<BoundCopy>(
-        CopyDescription(boundFilePaths, csvReaderConfig, actualFileType), tableID, tableName);
+        CopyDescription(direction, boundFilePaths, csvReaderConfig, actualFileType), tableID,
```

Comment:
I think we can have following design for `BoundCopy`. Let's start with the simplified version of only CopyTo node table

```
struct BoundCopyFromInfo {
     string tableName;
     share_ptr<NodeExpression> node
}

struct BoundCopyToInfo {
     share_ptr<NodeOrRelExpression> node
     expression_vector expressionsToCopy
}

class BoundCopy {

private:
    CopyDescription
    unique_ptr<BoundCopyFromInfo>
    unique_ptr<BoundCopyToInfo>
}
```

So you need to create a node Expression through `binder.createQueryNode()` and loop through all node properties through `node.getPropertyExpressions` to construct a `expressionsToCopy`

Reply:
What do you think about these changes?
1. Move `CopyDirection` to `BoundCopy`
2. Add `CopyDescriptionFrom` and `CopyDescriptionTo` to separate the logic of each one. For example, the current `CopyDescription` contains `CSVReaderConfig`, which is not related to `COPY TO`, and also `filePaths` is a vector, which makes sense for `COPY FROM` but not for `COPY TO`. Inside `CopyDescriptionXXX` (or maybe call `BoundCopyXXXInfo`?) we add the specific properties for each one.

Reply:
This design looks good to me.